### PR TITLE
Nojira | @ericbakenhus | Fix event import URL

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Fix import function URL in scheduled function
- Trigger an update if region is missing on existing events

# Review By (Date)
- ASAP/Retro

# Criticality
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code

### CODE REVIEW

File: src/utilities/synchronizedEvents.js [hunk 1]
```diff
@@ -77,20 +77,30 @@ export const mergeEventOverrides = (eventContent) => {
...
+
+export const setStoryRegion = async (story, mapKey) => {
+  const updatedStory = { ...story };
+  const { region, latitude, longitude } = updatedStory.content;
+
+  if (region || !latitude || !longitude) {
+    return updatedStory;
+  }
+
+  if (!storyHasValidLatLong(updatedStory)) {
```
Comment (ericbakenhus) [inline src/utilities/synchronizedEvents.js:103]: Weird diff. Just breaking out the lat/long check into it's own `storyHasValidLatLong` function.

File: netlify/functions/event-import-background.mts [hunk 2]
```diff
@@ -264,7 +266,7 @@ export default async (req: Request) => {
             continue;
           }
 
-          if (!compareStoryContent(google.content, storyblok.content)) {
+          if (!isMissingRegion && !compareStoryContent(google.content, storyblok.content)) {
```
Comment (ericbakenhus) [inline netlify/functions/event-import-background.mts:269]: Trigger an update attempt if region is missing but lat/long exists for existing events.

File: netlify/functions/trigger-event-import.mts [hunk 3]
```diff
@@ -12,10 +12,9 @@ export default async (req: Request) => {
       throw new Error('No secret available');
     }
 
-    const url = new URL('/webhook/sb/import-events', req.url);
+    const url = new URL('/webhook/sb/event-import', req.url);
```
Comment (ericbakenhus) [inline netlify/functions/trigger-event-import.mts:15]: Dangit. 🤦

File: netlify/functions/trigger-event-import.mts [hunk 4]
```diff
@@ -33,5 +32,5 @@ export default async (req: Request) => {
 };
 
 export const config: Config = {
-  schedule: '0 15 * * *', // Every day at 3 PM UTC (8 AM PT)
+  schedule: '30 15 * * *', // Every day at 3:30 PM UTC (7:30 PST/8:30 PDT)
```
Comment (ericbakenhus) [inline netlify/functions/trigger-event-import.mts:35]: Push the schedule by 30 min to make sure we're triggering after the sheet update regardless of any DST weirdness.

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**